bodo/tests/iceberg_database_helpers/utils.py

```diff
@@ -106,14 +106,6 @@ def do_get_spark():
     return do_get_spark()
 
 
-def get_spark_tabular(tabular_connection, path="."):
-    if bodo.get_rank() != 0:
-        return None
-    spark = get_spark(path=path)
-    spark.sql("use default;")
```

Comment:
@IsaacWarren do you know why "use default" was necessary in the case of tabular?